src/test/unit/algorithms/SpatialPoolerTest.cpp

```diff
@@ -879,7 +879,7 @@ TEST(SpatialPoolerTest, testBumpUpWeakColumns) {
       //  Inc   -     -       -     Inc   Inc     -     Inc
       {0.000, 0.000, 0.074, 0.000, 0.062, 0.054, 0.110, 0.000}, // unchanged
       //  -      -      -      -     -      -      -      -
-      {0.061, 0.010, 0.010, 0.000, 0.000, 0.000, 0.188, 0.000},
+      {0.061, 0.011, 0.011, 0.000, 0.000, 0.000, 0.188, 0.000},
```

Comment:
this is because of the raisePermanenceTrim is removed, right?

Reply:
> this is because of the raisePermanenceTrim is removed, right?

No.  Its because the method `SP::setPermanence()` is badly written.  I will rewrite and change this test back.